src/modules/75-cf/constants/GitSyncConstants.ts

```diff
@@ -5,7 +5,7 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-export const AUTO_COMMIT_MESSAGES: Record<string, string> = {
+export const GIT_COMMIT_MESSAGES: Record<string, string> = {
```

Comment:
It would be awesome if we can move these messages to the strings file. This can then be the keys of the strings and when you go to use it, pass it through `getString`.
```suggestion
export const GIT_COMMIT_MESSAGES: Record<string, StringKeys> = {
```

Reply:
These messages are likely to change when we move to Git X, we'll use whatever patterns/messages they have for us in that case. So I'm inclined to leave this alone for now since it'll change anyway